docs/book/getting-started/zenml-pro/pro-api.md

```diff
@@ -0,0 +1,73 @@
+---
+description: >
+  Learn about how to use the ZenML Pro API programmatically.
+---
+
+# Using the ZenML Pro API
+
+ZenML Pro offers a powerful API that allows you to interact with your ZenML resources. Whether you're using the [SaaS version](https://cloud.zenml.io) or a self-hosted ZenML Pro instance, you can leverage this API to manage tenants, organizations, users, roles, and more.
+
+The SaaS version of ZenML Pro API is hosted [here](https://cloudapi.zenml.io).
+
+## API Overview
+
+The ZenML Pro API is a RESTful API that follows OpenAPI 3.1.0 specifications. It provides endpoints for various resources and operations, including:
+
+- Tenant management
+- Organization management
+- User management
+- Role-based access control (RBAC)
+- Authentication and authorization
+
+## Authentication
+
+To use the ZenML Pro API, you need to authenticate your requests. If you are logged in to your ZenML Pro account,
+you can use the same browser window to authenticate requests to your ZenML Pro API, directly in the OpenAPI docs. 
+
+For example, for the SaaS variant, you can access the docs here: https://cloudapi.zenml.io. You can make requests
+by being logged into ZenML Pro at https://cloud.zenml.io.
+
+Programmatic access is not possible at the moment.
```

Comment:
confusing. you say at the top that this page is about programmatic access to the API, but here you say it isn't.